VIDEO_GET_FRAME_RATE is removed with linux-libc-headers 4.19.
Define VIDEO_GET_FRAME_RATE, based on:
https://elixir.bootlin.com/linux/v3.2/source/include/linux/dvb/video.h#L255

diff --git a/main.c b/main.c
index 55c4fa3..ecc27eb 100644
--- a/main.c
+++ b/main.c
@@ -34,6 +34,7 @@
 
 #define Debug(level, str, ...) (DebugLevel > level ? printf("LIBDVD: %07.3f: " str, (float) ddvd_get_time() / 1000.0, ##__VA_ARGS__) : 0)
 #define Perror(msg)            Debug(-1, "%s: %s", msg, strerror(errno))
+#define VIDEO_GET_FRAME_RATE   _IOR('o', 56, unsigned int)
 
 int DebugLevel = 1;
